parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -0,0 +1,355 @@
+use super::{
```

Comment:
Some files like this one missing general comment with licence info.

Reply:
good spot. Have fixed that and added a few module level doc lines.